docs/dev/framework/image-processing/image-studio.md

```diff
@@ -136,23 +136,69 @@ This can be helpful if you need to output multiple images with a similar configu
 
 ### PHP Templates
 
-If you are creating your own PHP templates and want to apply `Figure` data directly, you currently need to create the
-markup yourself.
+There are two ways to output images in your PHP templates:
 
-If you are instead using the legacy Contao `image.html5` template, you can make use of two builtin helper functions to
-apply/get an array of data ready to use. 
+1) **Rendering a `Figure`** &mdash; If you are creating your own PHP templates and want to apply `Figure` data
+   directly, you currently need to create the markup yourself.
 
-```php
-$template = new FrontendTemplate('image');
+   If you are instead using the legacy Contao `image.html5` template, you can make use of two builtin helper functions
```

Comment:
Why is the `image.html5` template legacy?

Reply:
I didn't change that part. :grin:

Reply:
Right :D